ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -427,6 +447,18 @@ export default class TestDataModal extends React.Component {
       },
     ];
 
+    if (tags.length > 1) {
+      modalOptions = [
+        ...modalOptions,
+        {
+          options: tags.sort(),
+          selectedItem: tag,
```

Comment:
It'd be desirable to show the default tag as "all tags", similar to "all status" & "all frameworks" from Alerts view.